tensorboard/components/tf_ng_tensorboard/plugins/plugins.component.ts

```diff
@@ -57,8 +74,28 @@ export class PluginsComponent implements OnInit {
     // We manually create plugin DOM (with custom tagName and script inside
     // an iframe) when the `activePlugin` changes.
     this.activePlugin$
-      .pipe(filter(Boolean))
-      .subscribe((plugin) => this.renderPlugin(plugin as UiPluginMetadata));
+      .pipe(
+        filter(Boolean),
```

Comment:
You are correct, but as a view, I don't want to make such assumptions about the how store initialization works. We just know that it is null-able and we are not interested in that case.

Reply:
My point is not about the store initialization, except to the extent that it's the reason for making activePlugin$ nullable at all.  As this stands, if either the plugin list or the active plugin ID became null at some point, renderPlugin() would not be called and so the previously active plugin would remain visible (hence out of sync with the state).  IIUC this is not a concern because, once the plugin list and the active id are set to a real value, they should never revert to null.  Right?